lib/SIL/SILVerifier.cpp

```diff
@@ -108,6 +108,7 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
   SILFunctionConventions fnConv;
   Lowering::TypeConverter &TC;
   SILOpenedArchetypesTracker OpenedArchetypes;
+  SmallVector<SILDebugVariable, 16> DebugVars;
```

Comment:
But I thought it runs after each SIL pass? In any case switching it over to just using the name shouldn't be much work and then at least we get rid of the redundant ArgNo :-)